migrations/v1_3_0_configs.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
Why have you named those files 1_3_0?
We are nowhere near version 1.3.0 :P Also we haven't released the other 1_2_0 migrations files yet, so I would suggest to add this stuff there, both config and schema, cause it is connected.